### PR TITLE
analog: rework SetParameter fromBytes

### PR DESCRIPTION
**Pull request by cleresd**

### CODE REVIEW

File: src/analog/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -1721,6 +1721,15 @@ class CommandBinaryBuffer extends BinaryBuffer {
         this.setUint16(parameter.resistanceStopThreshold, false);
     }
 
+    private getMqttSessionConfig (): IParameterMqttSessionConfig {
```
Comment (DeadByElpy) [inline src/analog/CommandBinaryBuffer.ts:1724]:
```suggestion
    private getMqttSessionConfigParameter (): IParameterMqttSessionConfig {
```

and 'setMqttSessionConfig' -> 'setMqttSessionConfigParameter' too

File: src/analog/CommandBinaryBuffer.ts [hunk 2]
```diff
@@ -1800,6 +1809,14 @@ class CommandBinaryBuffer extends BinaryBuffer {
         this.setUint8(parameter.version);
     }
 
+    private getNbiotSslWrite (): IParameterNbiotSslWrite {
```
Comment (DeadByElpy) [inline src/analog/CommandBinaryBuffer.ts:1812]:
```suggestion
    private getNbiotSslWriteParameter (): IParameterNbiotSslWrite {
```
and in other places like this

File: src/analog/commands/downlink/SetParameter.ts [hunk 3]
```diff
@@ -159,6 +159,197 @@ const examples: TCommandExampleList = [
...
+            id: deviceParameters.MQTT_BROKER_ADDRESS,
+            data: {
+                hostName: 'chirpstack-example.com',
+                port: 1800
+            }
+        },
+        hex: {header: '03 1a', body: '23 16 63 68 69 72 70 73 74 61 63 6b 2d 65 78 61 6d 70 6c 65 2e 63 6f 6d 07 08'}
+    },
+    {
+        name: 'Enable ssl',
```
Comment (DarkPark) [inline src/analog/commands/downlink/SetParameter.ts:188]:
```suggestion
        name: 'enable ssl',
```

File: src/analog/commands/downlink/SetParameter.ts [hunk 4]
```diff
@@ -159,6 +159,197 @@ const examples: TCommandExampleList = [
             }
         },
         hex: {header: '03 05', body: '21 8c a0 65 90'}
+    },
+    {
+        name: 'set configuration for session',
```
Comment (DarkPark) [inline src/analog/commands/downlink/SetParameter.ts:164]: somewhere in name should be mqtt

and in next examples

File: src/analog/CommandBinaryBuffer.ts (hunk 1 above)
Comment (cleresd) [reply src/analog/CommandBinaryBuffer.ts]: ++

File: src/analog/CommandBinaryBuffer.ts (hunk 2 above)
Comment (cleresd) [reply src/analog/CommandBinaryBuffer.ts]: ++

File: src/analog/commands/downlink/SetParameter.ts (hunk 3 above)
Comment (cleresd) [reply src/analog/commands/downlink/SetParameter.ts]: ++

File: src/analog/commands/downlink/SetParameter.ts (hunk 4 above)
Comment (cleresd) [reply src/analog/commands/downlink/SetParameter.ts]: ++

**Review by DarkPark — APPROVED** [review]

**Pull request merged**